server/models/blogModel.js

```diff
@@ -1,18 +1,23 @@
-const mongoose = require('mongoose');
-const Comment = require('../models/commentModel');
+const mongoose = require("mongoose");
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using ES6 import statement</b></div>
<div id="fix">

Consider using ES6 `import` statement instead of `require()`. This helps maintain consistency with modern JavaScript practices and enables better static analysis.

Suggested change:
Replace `const mongoose = require('mongoose')` with `import mongoose from 'mongoose'`
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
import mongoose from "mongoose";
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Divyanshu9822/blog-mern-app/pull/22#issuecomment-2544587479>#245782</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged